tests/phpunit/tests/block-supports/typography.php

```diff
@@ -359,7 +360,11 @@ public function data_generate_font_size_preset_fixtures() {
 				'font_size_preset' => array(
 					'size' => null,
 				),
-				'settings'         => null,
+				'settings'         => array(
+					'typography' => array(
```

Comment:
Addressing an omission - here the tests is checking whether a `null` value is returned even when fluid typography is enabled.